src/components/RegistrationList.tsx

```diff
@@ -134,6 +161,25 @@ export const RegistrationListItemContent = ({
 
       {canEditRegistration && (
         <Box sx={{ display: 'flex', alignItems: 'start', gap: '0.5rem' }}>
+          {location.pathname.includes(UrlPathTemplate.MyPageMyResults) && (
```

Comment:
Kan denne sjekke for eksakt likhet? Da vil det fortsette likt ved eventuelle utvidelser med flere parametre også
```suggestion
          {location.pathname === UrlPathTemplate.MyPageMyResults && (
```